backend/prisma/seed.ts

```diff
@@ -28,6 +30,25 @@ async function main(): Promise<void> {
       data: user_security,
     });
   }
+  for (const region of regions) {
+    const dataToSeed = new AutoriaPlainDataDto(region);
+    const newRegion = await prisma.region.create({
+      data: dataToSeed,
+    });
+    const city = cities.filter(
+      (city: { data: { name: string; value: number }[]; regionId: number }) =>
+        city.regionId === region.value,
+    );
+    for (const data of city[0].data) {
```

Comment:
There's a method `.find()` in JS which returns the first element that matched the search condition

```suggestion
    const city = cities.find(
      (city: { data: { name: string; value: number }[]; regionId: number }) =>
        city.regionId === region.value,
    );
    for (const data of city.data) {
```

Reply:
Ok, but then we need to cast type or check if `city` exists. Otherwise it will throw `TypeError: Object is possibly 'undefined'.`